src/entities/Article.ts

```diff
@@ -0,0 +1,34 @@
+import { Field, ID, ObjectType } from 'type-graphql';
+import { prop as Property, getModelForClass } from '@typegoose/typegoose';
+
+@ObjectType({ description: 'The Article Model' })
+export class Article {
+  @Field(() => ID)
+  id: string;
+
+  @Field()
+  @Property()
+  title: string;
+
+  @Field()
+  @Property()
+  publication: string;
```

Comment:
i've never done a TS backend project so maybe I just don't know the syntax, but shouldn't this be a FK to reference the other Publication entity?

Reply:
Not with the current design because Publication isn’t a table being stored in the database. However, I can change this to being a MongoDB entity and adding a new table for publishers. The motivation behind the design was that publisher data is static, so there wasn’t too much of a need to make a table for it. This isn’t really scalable though for more features in the future so it may be worth changing.

Reply:
@orkosinha let's keep what you have so far and make a github issue for it to change it after this PR is merged tbh, as long as we're aware there's a more scalable solution